.github/workflows/go.yml

```diff
@@ -0,0 +1,46 @@
+---
+
+name: Go
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main, develop]
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+jobs:
+
+  build:
+    name: Build on ${{ matrix.os }} with Go ${{ matrix.go }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [macos-latest, windows-latest, ubuntu-latest]
+        go: ['1.15', '1.14']
```

Comment:
Please add `1.16` and remove `1.14`. The official supported Golang versions are always only "the current one" and "the previous one".